src/App.js

```diff
@@ -62,11 +52,12 @@ export default function BasicDatePicker() {
                   //control={<Switch defaultChecked />}
                   control={<Switch />}
                   label="Is it international flight?"
+                  required
                 />
               </FormGroup>{" "}
             </div>
             <div className="d-grid gap-2 mt-3">
-              <button type="submit" className="btn btn-dark">
+              <button type="submit" className="btn btn-dark" onClick={() => {alert("Informations saved.")}}>
```

Comment:
Let's not use an alert for now as it blocks the whole window from the user which is not so good for user experience. Please refer [here](https://ux.stackexchange.com/a/4522) for the reasonings behind. So maybe we can show a message within the app itself below / above  the form. wdyt @calovey ?